### PR TITLE
Add settings to change workday hours

### PR DESCRIPTION
**Pull request by jtexeira**
Added settings screen, can be accessed by pressing '?'.
This menu was made by reusing the 'Edit Action' popup.
It takes values like '8.6'.

Settings are saved on '<base-file-name>-config', in json format.

### CODE REVIEW

File: src/app.rs [hunk 1]
```diff
@@ -225,6 +250,8 @@ impl App {
     pub fn save_to<P: AsRef<Path>>(&self, p: P) -> io::Result<()> {
         let acts = self.activities.iter().flat_map(|(_, acts)| acts.iter());
         File::create(p.as_ref()).and_then(|f| store_activities(f, acts))?;
+        File::create(format!("{}-config", p.as_ref().display()))
```
Comment (mendess) [inline src/app.rs]: wouldn't it be better to save to just `config` instead? Otherwise there will be a different config per month

**Review by mendess — CHANGES_REQUESTED** [review]

File: src/app.rs (hunk 1 above)
Comment (jtexeira) [reply src/app.rs]: Ye, haven't thought of that really. It was just to make it easier to store the config file in a custom place.
Maybe just store it to the ~/.config location

Comment (mendess) [reply src/app.rs]: sure, use the [dirs](https://docs.rs/dirs/latest/dirs/) crate

File: src/app.rs [hunk 2]
```diff
@@ -54,18 +55,16 @@ impl App {
...
 
-    pub fn new(
-        filename: String,
-        activities: Vec<Activity>,
-        days_off: Vec<Date>,
-        config: Config,
-    ) -> Self {
+    pub fn new(filename: String, activities: Vec<Activity>, days_off: Vec<Date>) -> Self {
+        let mut conf_path = dirs::config_dir().unwrap();
+        conf_path.push("effort_config");
```
Comment (mendess) [inline src/app.rs]: `effortrc`?

**Review by mendess — APPROVED** [review]

File: src/app.rs (hunk 2 above)
Comment (jtexeira) [reply src/app.rs]: Good idea :clap:

**Pull request merged**